sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListModel.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.sample.ui.surveylist
+
+import co.nimblehq.survey.sdk.entity.SurveyEntity
+
+/**
+ * Survey List that is exposed to the UI
+ */
+data class SurveyListModel(
+    val listSurvey: List<SurveyEntity>,
```

Comment:
- I feel like this is a bit redundant 🤔  we wrap a class name List over a single attribute that is an actual `List` 🤔  
- Another catch: the View now has the knowledge of `SurveyEntity` - which is sent from the ViewModel that retrieved from the 3rd-party API: our survey-sdk. It would be best if we convert it to just include what you wanted to display and don't let the View knows too much. For example, if you plan to display only the `title` - then can we just have:

```
val surveyList = LiveData<SurveyUiModel<List<SurveyModel>>> 

data class SurveyModel {
   val title: String
}

//from View:

surveyList.observe { setAdapter( it.data ) } 

//RecyclerView Placeholder:
TextView.bind(data.title)
```

just some psuedo code 🙏  let me know if you need further clarification

Reply:
@sleepylee 
1. Maybe we don't need the model here. Just use the list to make it simple
2. You're right, in the real project, we need to map the `Entity` to `Model`.
But in this circumstance , I want to make more simple, by using the `entity` directly to reduce the effort on `sample-app`. Just make sure the `sample-app` can get data from `sdk`
What do you think?

Reply:
we will make more updates to the sdk later, so just want to make sure that you understood if we separate the concern early on, later you can change the code in the SDK without changing anything in the `sample-app` 🙋

Reply:
@sleepylee I've updated the code, please help to review if you have spare time.